docs/Calendar/BookingsDiagnosticSummary.md

```diff
@@ -0,0 +1,50 @@
+# BookingsDiagnosticSummary
+
+Download the latest release: [BookingsDiagnostic.ps1](https://github.com/microsoft/CSS-Exchange/releases/latest/download/BookingsDiagnosticObjectsSummary.ps1)
+
+This script runs a series of tests in a bookings Mailbox (one per execution) and returns a summarized list of the bookings Mailbox characteristics, as well as testing for known configuration issues that can lead to bookings not performing as expected.
+
+Additionally, it will collect the most common logs needed for troubleshooting by support, including:
+
+* Staff Membership log
+* Message Tracing Log
+* Booking Mailbox configuration
+* Staff List and Permissions
+* Services configuration
+
+To run the script, you will need a valid SMTP Address for a booking Mailbox.
+
+| Parameters:                   | Explanation:                                                                                                                                                                                                                                                                                                                                                                      |
+| :---------------------------- | :-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| **-Identity**           | Booking MB SMTP Address (Only one per execution)                                                                                                                                                                                                                                                                                                                                  |
+| **-Staff**              | Verify Staff permissions for the Bookings mailbox.                                                                                                                                                                                                                                                                                                                                |
```

Comment:
Here or above, do you want to state default is to run all tests?
i.e. for here something like:
| **-Staff** | Verify Staff permissions for the Bookings mailbox. (default is to run Staff verification)

Reply:
added in the description all parameters but exception are optional and default to true (run all tests for a MB)